fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/Correlated.java

```diff
@@ -232,5 +232,10 @@ public boolean equals(final Object o) {
         public int hashCode() {
             return Objects.hash(aliasMap);
         }
+
+        @Override
+        public String toString() {
+            return "Equivalence Bound to " + aliasMap;
```

Comment:
I am open for suggestion and rather not _just_ delegate to `aliasMap.toString()` to avoid potential confusion when debugging, we should indicate that this equivalence is relying somehow on an alias map to check for equality, e.g. in the context of membership checks in a hash map of `Value`s.

Reply:
`"≌[" + aliasMap + "]"`

Reply:
done.